src/OrleansRuntime/Messaging/SiloMessageSender.cs

```diff
@@ -31,9 +31,9 @@ namespace Orleans.Runtime.Messaging
 {
     internal class SiloMessageSender : OutgoingMessageSender
     {
-        private readonly MessageCenter messageCenter;
+        protected readonly MessageCenter messageCenter;
         private const int DEFAULT_MAX_RETRIES = 0;
-        private readonly Dictionary<SiloAddress, DateTime> lastConnectionFailure;
+        protected readonly Dictionary<SiloAddress, DateTime> lastConnectionFailure;
```

Comment:
Good catch, I agree it should be private.  I think this was just left over from some code that no longer exists.